### PR TITLE
Set version to 0.8.20

### PR DESCRIPTION
**Pull request by nikola-matic**

### CODE REVIEW

File: Changelog.md
Comment (nikola-matic) [inline Changelog.md:22]: This is just a cherry-pick from the other PR so we don't have to wait until its merged.

**Review by ekpyron — APPROVED** [review]

**Pull request merged**
